tasks/config/intern.js

```diff
@@ -8,6 +8,12 @@ module.exports = function (grunt) {
       config: 'test/intern',
       reporters: ['Console']
     },
-    dev: {}
+    dev: {},
+    api: {
```

Comment:
We talked about this previously, but I really don't think we need intern/an entire browser for these tests. If you want to do that in a separate PR I'm fine with it.